core/sequencer/server.go

```diff
@@ -186,16 +186,15 @@ func NewServer(
 }
 
 func (s *Server) UpdateMetrics(ctx context.Context, in *spb.UpdateMetricsRequest) (*spb.UpdateMetricsResponse, error) {
-	if err := s.unappliedMetric(ctx, in.DirectoryId); err != nil {
+	if err := s.unappliedMetric(ctx, in.DirectoryId, in.MaxUnappliedCount); err != nil {
```

Comment:
Should `MaxUnappliedCount` be populated somewhere? Otherwise `UpdateMetrics` will read 0.

Reply:
Thanks for catching that. This should be set in the sequencer server